apps/web/package.json

```diff
@@ -6,21 +6,27 @@
   "license": "Apache-2.0",
   "author": "Hylo <hello@hylo.com>",
   "type": "module",
+  "engines": {
+    "node": "^20",
+    "yarn": "^4.5.0"
+  },
+  "packageManager": "yarn@4.5.0",
   "browserslist": [
     "defaults"
   ],
   "proxy": "localhost:3001",
   "scripts": {
     "viteDev": "vite serve",
     "dev": "yarn start",
-    "start": "rm -rf ../../node_modules/.cache/babel-loader && node --preserve-symlinks scripts/start.js",
```

Comment:
`rm -rf ../../node_modules/.cache/babel-loader` was needed in the past due to a bug in graqphql parsing which meant that changes to existing files would not be reflected even after a restart. Do you know if this cleared-up?